js/apps/admin-ui/src/user/UsersSection.tsx

```diff
@@ -236,59 +306,425 @@ export default function UsersSection() {
   //should *only* list users when no user federation is configured
   const listUsers = !(userStorage.length > 0);
 
-  const toolbar = (
-    <>
+  const onSearch = () => {
+    if (searchValue !== "") {
+      setSearchValue(searchValue);
+    } else {
+      setSearchValue("");
+    }
```

Comment:
This condition is essentially moot as the following code does the exact same:
```suggestion
    setSearchValue(searchValue);
```

Reply:
Fixed